docs/content/preview/api/ysql/user-defined-subprograms-and-anon-blocks/language-plpgsql-subprograms/plpgsql-syntax-and-semantics/executable-section/loop-exit-continue-statements/_index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Loop, exit, and continue statements [YSQL]
+headerTitle: Loop, exit, and continue statements
+linkTitle: Loop, exit, and continue statements
+description: Describes the syntax and semantics of the various PL/pgSQL Loop statements. [YSQL]
+image: /images/section_icons/api/subsection.png
+menu:
+  preview:
+    identifier: loop-exit-continue-statements
+    parent: executable-section
+    weight: 40
+type: indexpage
+showRightNav: true
+---
+
+## Syntax
+
+<ul class="nav nav-tabs nav-tabs-yb">
+  <li >
+    <a href="#grammar-3" class="nav-link" id="grammar-tab" data-toggle="tab" role="tab" aria-controls="grammar" aria-selected="true">
+      <img src="/icons/file-lines.svg" alt="Grammar Icon">
+      Grammar
+    </a>
+  </li>
+  <li>
+    <a href="#diagram-3" class="nav-link active" id="diagram-tab" data-toggle="tab" role="tab" aria-controls="diagram" aria-selected="false">
+      <img src="/icons/diagram.svg" alt="Diagram Icon">
+      Diagram
+    </a>
+  </li>
+</ul>
+
+<div class="tab-content">
+  <div id="grammar-3" class="tab-pane fade" role="tabpanel" aria-labelledby="grammar-tab">
```

Comment:
There are two diagram-pairs in this page. The first pair has these rules:

```
plpgsql_loop_stmt,plpgsql_unbounded_loop_defn,plpgsql_bounded_loop_defn,plpgsql_integer_for_loop_defn,plpgsql_array_foreach_loop_defn,plpgsql_query_for_loop_defn,plpgsql_dynamic_subquery
```

And the second pair has these rules:

```
plpgsql_exit_stmt,plpgsql_continue_stmt
```

When a page has two or more diagram pairs, then the convention is that the first uses plain names like _\<a href="#grammar" ..._, the second adds _-2_ to get _\<a href="#grammar-2" ..._, and so on.

I implemented that convention now. I had it wrong before. I don't know what it's about. Nobody has explained it to me. If it matters, then it should be explained in **[Edit syntax diagrams](https://docs.yugabyte.com/preview/contribute/docs/syntax-diagrams/)**.

Anyway, both diagram-pairs change in lock-step when you select _either_ "Grammar" _or_ "Diagram" on any one of the pairs. This is what I want.

Reply:
The convention - that each set of tabs on a page have different IDs/href - is required for the tab switching code to function properly, otherwise the latter tabs don't work properly. This is not specific to syntax diagrams - it applies anywhere there are multiple sets of tabs on a page. Possibly Widgets and Shortcodes page.